src/epub/text/chapter-11.xhtml

```diff
@@ -85,7 +85,7 @@
 			<p>She had finished her breakfast, so I permitted her to give a specimen of her accomplishments. Descending from her chair, she came and placed herself on my knee; then, folding her little hands demurely before her, shaking back her curls and lifting her eyes to the ceiling, she commenced singing a song from some opera. It was the strain of a forsaken lady, who, after bewailing the perfidy of her lover, calls pride to her aid; desires her attendant to deck her in her brightest jewels and richest robes, and resolves to meet the false one that night at a ball, and prove to him, by the gaiety of her demeanour, how little his desertion has affected her.</p>
 			<p>The subject seemed strangely chosen for an infant singer; but I suppose the point of the exhibition lay in hearing the notes of love and jealousy warbled with the lisp of childhood; and in very bad taste that point was: at least I thought so.</p>
 			<p>Adèle sang the canzonette tunefully enough, and with the naivete of her age. This achieved, she jumped from my knee and said, “Now, Mademoiselle, I will repeat you some poetry.”</p>
-			<p>Assuming an attitude, she began “La Ligue des Rats: fable de La Fontaine.” She then declaimed the little piece with an attention to punctuation and emphasis, a flexibility of voice and an appropriateness of gesture, very unusual indeed at her age, and which proved she had been carefully trained.</p>
+			<p>Assuming an attitude, she began “<span epub:type="se:name.publication.poem" xml:lang="fr">La Ligue des Rats</span>: <i xml:lang="fr">fable de La Fontaine</i>.” She then declaimed the little piece with an attention to punctuation and emphasis, a flexibility of voice and an appropriateness of gesture, very unusual indeed at her age, and which proved she had been carefully trained.</p>
```

Comment:
If `fable de La Fontaine` is a subtitle, then it should not be italicized. Titles in non-English don't get italics.

Reply:
It’s not part of the official poem title in any way. It seems to be there to tell the reader where the poem is from.

Reply:
I think it is a type of subtitle, see https://www.google.com/books/edition/Fables_de_La_Fontaine/mbABAAAAYAAJ?hl=en&gbpv=1&dq=%22La+Ligue+des+rats%22&pg=PA283&printsec=frontcover

Reply:
Oops, this link rather: https://www.google.com/books/edition/Fables_de_La_Fontaine/mbABAAAAYAAJ?hl=en&gbpv=1&pg=PR3&printsec=frontcover

Reply:
Fair enough. It isn’t part of the official title, but if “a type of” subtitle works for you, it certainly works for me.